src/SemanticLog/Logger.php

```
@@ -268,22 +270,62 @@ private function extractProperties(object $result): array
+        return $properties;
+    }
+
+    /**
+     * @return ObjectProperties
+     * @phpstan-return array<string, mixed>
+     */
+    private function extractDynamicProperties(object $result): array
+    {
+        $rawProperties = get_object_vars($result);
+        $properties = [];
+        foreach ($rawProperties as $name => $_) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$_'. (undefined)

(UnusedLocalVariable)